README.md

```diff
@@ -98,6 +98,7 @@ Legend:
 | Number Field | [XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/eaa9ee19-4411-4648-b19d-41f61f9a01cf) |  [Issue](https://github.com/ni/nimble/issues/361) | [:white_check_mark: - SB](https://nimble.ni.dev/storybook/?path=/docs/components-number-field--docs) | :white_check_mark: | :white_check_mark: |
 | Progress Bar | |  [Issue](https://github.com/ni/nimble/issues/304) | :o: | :o: | :o: |
 | Radio | [XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/3698340b-8162-4e5d-bf7a-20194612b3a7) |  [Issue](https://github.com/ni/nimble/issues/297) | [:white_check_mark: - SB](https://nimble.ni.dev/storybook/?path=/docs/components-radio-group--docs) | :white_check_mark: | :white_check_mark: |
+| Rich Text Viewer | |  [Issue](https://github.com/ni/nimble/issues/1288) | [:warning: - SB](https://nimble.ni.dev/storybook/?path=/docs/incubating-rich-text-viewer--docs) | :warning: | :warning: |
```

Comment:
Heads up that the source for the status table is moving in #1312. Equivalent code will go in `packages/nimble-components/docs/component-status.stories.ts` once that's complete.